Makefile

```diff
@@ -11,4 +11,8 @@ run:
 	cd prototypes/form-wizard && \
 	npm run serve
 
-.PHONY: build setup run
+test:
+	cd prototypes/form-wizard && \
+	echo ok
```

Comment:
maybe this could be something more substantial, just to verify the build? like `stat _site/assets/js/agencies.js && echo ok`?